crypto/rsa/rsa_lib.c

```diff
@@ -188,15 +188,15 @@ int RSA_set0_key(RSA *r, BIGNUM *n, BIGNUM *e, BIGNUM *d)
         return 0;
 
     if (n != NULL) {
-        BN_free(r->n);
+        BN_clear_free(r->n);
```

Comment:
This value is public and doesn't need clearing.

Reply:
But in RSA_free() it cleans by BN_clear_free. Change it?

Reply:
Another thoughts on this is that it's always safe to clear the memory. But I am not sure if this is the proper context...

Reply:
But I agree with @paulidale , `n` and `e` seem not necessary to be cleared...

Reply:
I agree that if there is any doubt, memory should be cleared.
I'm not in doubt over `n` and `e`.

I'd change them in `RSA_free` as well.